examples/gke-security-best-practices/builder/platform-uploads/tmp/roles/cis/meta/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+allow_duplicates: no
+
+
+galaxy_info:
+  author: Gordon Young
+  # 2017 VGS - Gordon Young, gordon.young@verygoodsecurity.com
+  description: Configure Unbuntu 14.04 LTS Security based on CIS for Ubuntu Linux 14.04 LTS Benchmark v2.0.0 - 09-30-2016
+  min_ansible_version: 1.9.0.1
+  platforms:
+    - name: Ubuntu
+      versions:
+        14.04
+  categories:
+    #- cloud
```

Comment:
Why are these categories commented out?

Reply:
these roles are not used. I will address them

Reply:
removed in favor of a pre-build CIS security hardened image image from the GCP marketplace